app/views/records/edit_fields/_subject.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= f.input :subject,
```

Comment:
This will eventually need to be dynamic (to work for any flex defined controlled value).